router/core/request_context_fields.go

```diff
@@ -68,7 +86,15 @@ func NewDurationLogField(val time.Duration, attribute config.CustomAttribute) za
 	return zap.Skip()
 }
 
-func AccessLogsFieldHandler(attributes []config.CustomAttribute, err any, request *http.Request, responseHeader *http.Header) []zapcore.Field {
+func AccessLogsFieldHandler(
+	logger *zap.Logger,
+	attributes []config.CustomAttribute,
+	exprAttributes []requestlogger.ExpressionAttribute,
+	passedErr error,
```

Comment:
The previous type was "any" because it took inputs from panics and errors, I went ahead and split it into two as we want to maintain the error type in the `ExpressionContext`, but if we are ok to make it even more generic just for the panic, we can merge the fields again.